NitroxClient/Helpers/ThrottledPacketSender.cs

```diff
@@ -24,7 +24,7 @@ public void Update()
                     continue;
                 }
 
-                if (packetSender.Send(throttledPacket.Packet))
+                if (packetSender.SendIfGameCode(throttledPacket.Packet))
```

Comment:
Will this ever work since it always originates from our Update()?

Reply:
In order to not change logic I basically renamed `Send` to `SendIfGameCode`, keeping the original suppression check until it can be removed (see TODO in SendIfGameCode).